src/pdiagmat.jl

```diff
@@ -58,7 +61,8 @@ function \(a::PDiagMat, x::AbstractVecOrMat)
 end
 function /(x::AbstractVecOrMat, a::PDiagMat)
     @check_argdims a.dim == size(x, 2)
-    # return matrix for 1-element vectors `x`, consistent with LinearAlgebra
+    (VERSION >= v"1.9" && x isa AbstractVector) && return x ./ a.diag
```

Comment:
I thought there was a longer discussion about this change and it was decided to revert it in the end? Or maybe that was just something similar...

Reply:
In any case, could we improve the check and figure out the exact commit, and its version number, which changed this behaviour?

Reply:
I wasn't involved in/didn't see those discussions, but this is what I observed when testing on Julia 1.9.3.

Reply:
IIRC we use exact commits in most (all?) other such branches in PDMats.

BTW I had https://github.com/JuliaLang/julia/pull/49915 and the linked PR in mind.

Reply:
Okay, I reverted this change to bring it into compliance.